cli/build.rs

```diff
@@ -0,0 +1,103 @@
+use std::fs::{read, write};
+use std::path::PathBuf;
+use uuid::Uuid;
+
+use reqwest::blocking::Client;
+use std::env;
+use std::fs;
+
+/// This script downloads the schema if it's not in the file system (in a path that's .gitignored)
+/// If it exists, the script does nothing.
+/// If UPDATE_SCHEMA environment variable is set, we download the schema if the etag is different on the
```

Comment:
Can you also add this to the contributing guide?

Reply:
Yup!